decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -303,6 +306,75 @@ def translated_proposal_body_template
       def proposal_creation_params
         params[:proposal].merge(body_template: translated_proposal_body_template)
       end
+
+      def tabs
+        @tabs ||= items.map { |item| item.slice(:id, :text, :icon) }
+      end
+
+      def panels
+        @panels ||= items.map { |item| item.slice(:id, :method, :args) }
+      end
+
+      def items
+        @items ||= [
+          {
+            enabled: @proposal.linked_resources(:projects, "included_proposals").present?,
+            id: "included_projects",
+            text: t("decidim/budgets/project", scope: "activerecord.models", count: 2),
+            icon: resource_type_icon_key("Decidim::Budgets::Project"),
+            method: :cell,
+            args: ["decidim/linked_resources_for", @proposal, { type: :projects, link_name: "included_proposals" }]
+          },
+          {
+            enabled: @proposal.linked_resources(:results, "included_proposals").present?,
+            id: "included_results",
+            text: t("decidim/accountability/result", scope: "activerecord.models", count: 2),
+            icon: resource_type_icon_key("Decidim::Accountability::Result"),
+            method: :cell,
+            args: ["decidim/linked_resources_for", @proposal, { type: :results, link_name: "included_proposals" }]
+          },
+          {
+            enabled: @proposal.linked_resources(:meetings, "proposals_from_meeting").present?,
+            id: "included_meetings",
+            text: t("decidim/meetings/meeting", scope: "activerecord.models", count: 2),
+            icon: resource_type_icon_key("Decidim::Meetings::Meeting"),
+            method: :cell,
+            args: ["decidim/linked_resources_for", @proposal, { type: :meetings, link_name: "proposals_from_meeting" }]
+          },
+          {
+            enabled: @proposal.linked_resources(:proposals, "copied_from_component").present?,
+            id: "included_proposals",
+            text: t("decidim/proposals/proposal", scope: "activerecord.models", count: 2),
+            icon: resource_type_icon_key("Decidim::Proposals::Proposal"),
+            method: :cell,
+            args: ["decidim/linked_resources_for", @proposal, { type: :proposals, link_name: "copied_from_component" }]
+          },
+          # {
+          #   enabled: @proposal.public_participants.any?,
```

Comment:
Is this supposed to remain commented out?

Reply:
This piece of code is not required, I've removed the commented lines